### PR TITLE
Audio Metadata Server Pipeline

### PR DESCRIPTION
**Pull request by Waidhoferj**
## What's New?
While the audio files have been uploaded from the web app to G Drive for some time, the metadata wasn't being logged in the sql server. That's fixed now.

Also
- Added ``emphasis`` and ``script`` fields to AudioSampleMetadata
- Swapped ``filename`` with ``audio_file_id``, which will be far more useful for retrieving it from G Drive
- Added a couple of validations and formats to cover the new fields

## Type of change (pick-one)
- [x] Bug fix (_non-breaking change which fixes an issue_)
- [ ] New feature (_non-breaking change which adds functionality_)
- [ ] Breaking change (_fix or feature that would cause existing functionality to not work as expected_)
- [ ] This change requires a documentation update

## How Has This Been Tested?
I used postman to deliver a payload with a similar format as the example in the database_wrapper docstring. I also ensured that the formatted results propagated to the sequel server.

## Checklist (check-all-before-merge)
_formatting help: `- [x]` means "checked' and `- [ ]` means "unchecked"_

- [x] I documented my code according to the [Google Python Style Guide][1] (I used numpy docstring style guide, I hope thats okay)
- [x] I ran `./format.sh` because it automatically cleans my code for me 😄 (which is why like 27 files were touched)

### CODE REVIEW

**Comment by Waidhoferj** [comment]
Shoutout to @snekiam for figuring out that the settings.yml file was failing because it should be **settings.yaml**

**Review by snekiam — APPROVED** [review]
Looks good to me

**Comment by mfekadu** [comment]
>  I ran ./format.sh because it automatically cleans my code for me 😄 (which is why like 27 files were touched)


I'll approve this PR after we merge #121 first. I suspect some merge conflicts in our near future because `./format` ran there too. Although, maybe not since the `black` autoformatter is deterministic/has-same-outputs-every-time and thus might make this merge a lot smaller. `¯\_(ツ)_/¯` let's see what happens. 

Thanks for running it, I've been reluctant to do so for some time. It needed to happen

**Comment by Waidhoferj** [comment]
Just rebased from dev. Though Github is still mad for some reason, this branch should now be up to date.

**Comment by mfekadu** [comment]
@Waidhoferj 
Hmm... did you make sure to `git push` your changes? 

If so, then I suppose quickly fixing the conflicts through the GitHub UI should suffice. I’d change them myself but I think you have a better idea of the logic right now.

The conflicts seem small

File: flask_api.py
Comment (snekiam) [inline flask_api.py]: Is this a duplicate of the endpoint on line 171? They look the same to me.

**Comment by Waidhoferj** [comment]
@snekiam the entire path and function was duplicated 😅. Nice catch!

File: flask_api.py
Comment (Jason-Ku) [reply flask_api.py]: ^ bump, everything else looks perfect though

**Review by snekiam — APPROVED** [review]

**Comment by mfekadu** [comment]
oh wow. these conflicts are scary

**Comment by Waidhoferj** [comment]
Ik 😅. I'll handle it.

**Comment by Waidhoferj** [comment]
Farewell old friend, we knew ye well.

**Pull request closed without merging**
